README.md

```diff
@@ -0,0 +1,42 @@
+# EzPlatform I18N
+
+## Introduction
+
+This is a meta repository which centralize EzPlatform translations to ease synchronization with Crowdin.
+
+**Important** : This repository is not suppose to be used on a production project, each translation will be 
+splitted in a dedicated package.
+
+## Crowdin
+
+If you want to help us on translating EzPlatform feel free to contribute on [EzPlatform project on Crowdin][crowdin-ezplatform].
+
+### In context translation
+
+You can enable [Crowdin in context translation][in-context] by setting a specific cookie on a EzPlatform application.
+
+One way to do this is to open the development console (right click, inspect) and run these lines:
+
+- enable: document.cookie='ez_in_context_translation=1;path=/;'; location.reload();
+- disable: document.cookie='ez_in_context_translation=;expires=Mon, 05 Jul 2000 00:00:00 GMT;path=/;'; location.reload();
+
+## Repository split
+
+Each locale will be stored in it's own repository in [ezplatform-i18n organisation][ezplatform-i18n-org].
+
+This synchronisation is done using [dflydev/git-subsplit][dflydev] tool.
+
+    git subsplit init https://github.com/react-php/react
+    git subplit update
+    git subsplit publish "
+        fr_FR:git@github.com:ezplatform-i18n/ezplatform-i18n-fr_FR.git
+        ach_UG:git@github.com:ezplatform-i18n/ezplatform-i18n-ach_UG.git
+    " --heads=master
```

Comment:
Here we can have a list list that 

    --heads="master 2.0"

I guess we can set that during the next release